docs/openapi/schemas.yaml

```diff
@@ -29,6 +29,183 @@ URL:
+      type: string
+Config:
+  type: object
+  additionalProperties: true
+  properties:
+    slack:
+      description: Optional. The Slack configuration
+      $ref: '#/SlackConfig'
+    alerts:
+      description: Optional. The Alert configuration
+      $ref: '#/AlertsConfig'
+OrganizationConfig:
+  allOf:
+    - $ref: '#/Config'
+Organization:
+  type: object
+  properties:
+    id:
+      description: The organization ID in uuid format
+      $ref: '#/Id'
+    name:
+      description: The name of the organization
+      type: string
+    imsOrgId:
+      description: Optional. The ID of the Adobe IMS organization
+      type: string
+    config:
+      description: Optional. The configuration of the organization
+      $ref: '#/OrganizationConfig'
+    createdAt:
+      description: The date and time when the organization was created
+      $ref: '#/DateTime'
+    updatedAt:
+      description: The date and time when the organization was last updated
+      $ref: '#/DateTime'
+  example:
+    id: 'o1p2q3r4-s5t6-u7v8-w9x0-yz12x34y56z'
+    name: 'Example Organization'
+    imsOrgId: '1234567890ABCDEF12345678@AdobeOrg'
+    config:
+      slack:
+        channel: '#example-channel'
+        workspace: 'example-workspace'
+      alerts:
+        - type: '404'
+          byOrg: true
+          mentions:
+            - slack:
+                - '@user1'
+                - '@user2'
+            - slack:
+                - '@user3'
+                - '@user4'
+        - type: 'cwv'
+          byOrg: false
+          mentions:
+            - slack:
+                - '@user5'
+                - '@user6'
+    createdAt: '2023-12-15T09:30:00Z'
+    updatedAt: '2024-01-19T11:20:00Z'
+OrganizationList:
+  type: array
+  items:
+    $ref: './schemas.yaml#/Organization'
+OrganizationCreate:
+  type: object
+  required:
+    - name
+  properties:
+    name:
+      description: The name of the organization
+      type: string
+    imsOrgId:
+      description: Optional. The ID of the Adobe IMS organization
+      type: string
+      example:
+        - '1234567890ABCDEF12345678@AdobeOrg'
+    config:
+      description: Optional. The configuration of the organization
+      $ref: '#/OrganizationConfig'
+  example:
+    name: 'Example Organization'
+    imsOrgId: '1234567890ABCDEF12345678@AdobeOrg'
+    config:
+      slack:
+        channel: '#example-channel'
+        workspace: 'example-workspace'
+      alerts:
+        - type: '404'
+          byOrg: true
+          mentions:
+            - slack:
+                - '@user1'
+                - '@user2'
+            - slack:
+                - '@user3'
+                - '@user4'
+        - type: 'cwv'
+          byOrg: false
+          mentions:
+            - slack:
+                - '@user5'
+                - '@user6'
+OrganizationUpdate:
+  type: object
+  properties:
+    name:
+      description: The name of the organization
+      type: string
+    imsOrgId:
```

Comment:
Is there a case where the name or IMS Org ID for an organisation would need to be changed compared to the initial values?